docs/src/dev/build_config.md

```diff
@@ -28,6 +28,13 @@ within a repository using `yk` (including the `yk` repository itself). If you
 want to set/unset `YKB_YKLLVM_BIN_DIR` then `cargo clean` any repositories
 using `yk` before rebuilding them.
 
+## `YKB_TRACER`
+
+`YKB_TRACER` environment variable allows building yk with either `hwt` (Hardware Tracer) or `swt` (Software Software Tracer).
```

Comment:
Can we wrap lines to the same length as others in the file?

[Also probably start this sentence with "The".]